templates/email/subscription-item-added.hypr

```diff
@@ -0,0 +1,16 @@
+{% extends "email/email" %}
+{% block body-content %}
+
+<p>{{ labels.subscriptionActivatedBlob|string_format(siteContext.generalSettings.websiteName, domainName)|safe}}</p>
```

Comment:
use `subscriptionItemAddedBlob` key here instead of `subscriptionActivatedBlob`

Reply:
fixed the issue